java-shared-dependencies/dependency-analyzer/src/main/java/com/google/cloud/model/Relation.java

```diff
@@ -0,0 +1,7 @@
+package com.google.cloud.model;
+
+public enum Relation {
```

Comment:
Would you add javadoc? I guess this corresponds to some concept in deps.dev?

Reply:
Done.